fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
         update.extractall(self.workspaceunpack)
```

Comment:
Why is `workspaceunpack` not the unused root partition? This means that the current PR needs 3*(root_size) of free space available in the data partition.